src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const dayPrice = 40;
+  const discount1 = 20;
+  const discount2 = 50;
+  const total = dayPrice * days;
+
+  if (days < 3) {
+    return total;
```

Comment:
The condition to check for the minimum number of days to not receive a discount should use a constant that represents this threshold. For example, `const MIN_DAYS_FOR_DISCOUNT = 3;` and then `if (days < MIN_DAYS_FOR_DISCOUNT) { ... }`.